src/borg/_chunker.c

```diff
@@ -313,10 +313,20 @@ chunker_is_zero(const uint8_t *data, ssize_t length)
 
 static const int chunker_fast_zero = 0;
 
+/* performance (Haswell, E3-1231v3): in cache about ~16 GB/s, w/ real memory access about 12 GB/s */
+
 static int
 chunker_is_zero(const uint8_t *data, size_t length)
 {
-    for(; length; data++, length--) {
+    // TODO: alignment issues?
```

Comment:
in the current usage it shouldn't be a problem, since the beginning of a bytes object seems to be always word-aligned (I noticed that small bytes() are consistently aligned on odd 8 bytes on x64; larger ones are aligned to >=16 bytes), but it's something to easily trip over so the code should ultimately just deal with that.